Sources/Defaults.swift

```diff
@@ -115,7 +129,7 @@ extension UserDefaults {
 		}
 	}
 
-	private func isNativelySupportedType<T>(_ type: T.Type) -> Bool {
+	fileprivate static func isNativelySupportedType<T>(_ type: T.Type) -> Bool {
```

Comment:
I think I would just leave it as a private method but it doesn't really make a difference..

Reply:
Good catch. In an earlier iteration, I used it in the Observation file, and I forgot to change it back to private.